account-service/src/main/java/com/bdonor/accountservice/Repository/DynamoRepo.java

```diff
@@ -28,6 +36,58 @@
     @Autowired
 	private BCryptPasswordEncoder bCryptPasswordEncoder;
 
+    public boolean createUser(String bloodGroup, String firstName, String lastName, String email, String password, String addressline, String postcode){
+        User checkEmail = mapper.load(User.class, firstName, email);
+        if (checkEmail != null) {
+            System.out.println("User " + checkEmail.toString() + "Already exists!");
+            return false;
+        }
+
+        String URLink;
+        String[] LatLong = {};
+
+        try {
```

Comment:
https://github.com/we-3hackathon/bdonor/blob/6187d2ce7981e7af097dc7807764b3436d65d062/account-service/src/main/java/com/bdonor/accountservice/Repository/DynamoRepo.java#L46-L80

Reply:
PR35, A2 (nothing to worry about for now, future referencing only)